README.md

```diff
@@ -74,4 +77,177 @@ webView.apply {
 Here is a screenshot taken from a demo static html page that has syntax highlighting using Prism JS.
 
 | ![device-2020-07-18-092715](https://user-images.githubusercontent.com/99822/87853541-fc52d700-c8d8-11ea-9dc6-2d4c624f3b74.png) | ![device-2020-07-18-092727](https://user-images.githubusercontent.com/99822/87853542-fceb6d80-c8d8-11ea-9641-4ecf927b5a01.png) | ![device-2020-07-18-092736](https://user-images.githubusercontent.com/99822/87853543-fe1c9a80-c8d8-11ea-9e11-c9779202368e.png) |
-| --- | --- | --- |
\ No newline at end of file
+| --- | --- | --- |
+
+## Building your own Fragment or Custom View
+Ideally, there should be a modular component or custom-view that you **re-use** syntax highlighting with dynamic content.
+For that having a `Fragment` or custom `View` is ideal.
+
+We can taken the learning from [above](#under-the-hood) to wrap the library in fragment or custom view. 
+Both comes with advantage of it's own.
+Regardless if which option is chosen, the underlying code is _almost_ identical.
+
+### Custom View
+The advantage of custom view is that, it can be used in `Fragment` too. Let's take a look how we can templatize the HTML to load source code dynamically.
```

Comment:
~it can be used in `Fragment` too.~ 

it can be used anywhere, `Activity` or `Fragment`